### ISSUE TITLE
Multiple subscribers push notification Gateway timeout

### ISSUE THREAD

**Issue by LinasGr** [tpeczek/Demo.AspNetCore.PushNotifications#14]
Hi Tomash,
  I am trying to solve why notification goes to first subscriber only.  Push notification client hangs after pushing message to second subscriber, and throws an exception. Received HTTP response after 60671.4769ms - Gateway Timeout.To reproduce, just run the demo project and subscribe with Firefox and chrome. Than sending messages should produce 2 notifications. but only the first subscription in the database gets pushed.

**Comment by tpeczek** [comment]
Hi @LinasGr,

It's unlikely that this is related to the implementation. I (and as far as I know couple more) am using this project in production with much higher numbers of subscriptions.

In order to be thorough I've tested your exact scenario and it worked as expected (to actually see both notifications I had to change the code to add some random part to message because Windows communication center ignores identical messages).

The error you are receiving is suggesting a network issue. It's hard for me to speculate further about its nature.
